### DISCUSSION TITLE
how can i access my student pack benefits after i got approved ?

### DISCUSSION THREAD

**Discussion by messaouddhia** [community/community#118928] [GitHub Education]
### Select Topic Area

Question

### Body

it's been  3 days and i don't know how to access the benefits, please help and thanks.
![image](https://github.com/community/community/assets/63933319/0395a988-233b-4c2c-b4b1-047d8170ed79)

**Comment by Anders564** [comment]
Fuck

**Reply by Anders564** [reply]
Hhhhh
